src/lib.rs

```diff
@@ -13,7 +13,7 @@ extern crate hyper;
 extern crate router;
 extern crate urlencoded;
 extern crate redis;
-extern crate spaceapi;
+pub extern crate spaceapi;
 
 pub use spaceapi as api;
```

Comment:
It looks nicer, but can also be done on the client side, can't it? But it would be a breaking change if we remove it though.